code/modules/mob/living/simple_animal/humanoids/clown.dm

```diff
@@ -1,6 +1,7 @@
 /mob/living/simple_animal/hostile/clown
 	name = "clown"
 	desc = "A denizen of clown planet"
+	tt_desc = "E Homo sapiens corydon" //this is an actual clown, as opposed to someone dressed up as one
```

Comment:
Wouldn't this be Homo sapiens scurra?

Reply:
H.s. scurra is technically the "fancy" clown; H.s. corydon is the common clown and I think more appropriate for the in-game clown!

Reply:
ah thanks!